README.md

```diff
@@ -2,36 +2,40 @@
 Qubic Node Source Code - this repository contains the source code of a full qubic node.
 
 > MAIN (current version running qubic) <br>
-> [![EFIBuild](https://github.com/qubic-network/core/actions/workflows/efi-build-develop.yml/badge.svg?branch=main)](https://github.com/qubic-network/core/actions/workflows/efi-build-develop.yml)
+> [![EFIBuild](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml/badge.svg?branch=main)](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml)
 > <br>
 > DEVELOP (current version we're working on) <br>
-> [![EFIBuild](https://github.com/qubic-network/core/actions/workflows/efi-build-develop.yml/badge.svg?branch=develop)](https://github.com/qubic-network/core/actions/workflows/efi-build-develop.yml)
+> [![EFIBuild](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml/badge.svg?branch=develop)](https://github.com/qubic/core/actions/workflows/efi-build-develop.yml)
 
 ## Prerequisites
-To run a qubic node, you need the following parts:
-- Bare Metal Server/Computer with at least 8 Cores (high CPU frequency with AVX2 support)
-- At least 128GB memory
+To run a qubic node, you need the following spec:
+- Bare Metal Server/Computer with at least 8 Cores (high CPU frequency with AVX2 support) Check supported CPUs [here](https://www.epey.co.uk/cpu/e/YTozOntpOjUwOTc7YToxOntpOjA7czo2OiI0Mjg1NzUiO31pOjUwOTk7YToyOntpOjA7czoxOiI4IjtpOjE7czoyOiIzMiI7fWk6NTA4ODthOjY6e2k6MDtzOjY6IjQ1NjE1MCI7aToxO3M6NzoiMjM4Nzg2MSI7aToyO3M6NzoiMTkzOTE5OSI7aTozO3M6NzoiMTUwMjg4MyI7aTo0O3M6NzoiMjA2Nzk5MyI7aTo1O3M6NzoiMjE5OTc1OSI7fX1fYjowOw==/)
+- At least 128GB of storage
 - 1Gb/s synchronous internet connection
-- An USB Stick or SSD/HD attached to the Computer via USB
-- An UEFI Bios
+- A USB Stick or SSD/HD attached to the Computer via USB
+- An UEFI Bios 

```

Comment:
If it is "A USB", it should be also "A UEFI", right?

Reply:
the part about USB attached storage can be marked as optional. it is not anymore a requirement.

Reply:
Fixed

Reply:
Please change "A USB Stick or SSD/HD attached to the Computer via USB" to "A USB Stick or SSD/HD attached to the Computer (via NVMe M.2 or USB)"

Reply:
Done. I thought that was covered under USB Stick 'or'.